docs/build_node.md

```diff
@@ -503,9 +516,19 @@ The broader node-announcement message is not automatically broadcast.
 **Example:**
 
 ```java
-// TODO(matt) :)
+byte[] rgb = // insert the RGB "color" of this node
+byte[] alias = // insert the alias of this node, i.e. its human readable name
+NetAddress[] addresses = // insert a list of socket addresses on which this node
+                         // accepts incoming connections
+
+// Once per minute or so, call:
+channel_manager.broadcast_node_announcement(rgb, alias, addresses);
```

Comment:
This isn't really useful unless we can construct `NetAddress` objects, which we currently cannot.

Reply:
Ah okay. I'll remove the step for now, then?

Reply:
Yea, feel free to drop it, I'll fix it soon, though, hopefully.

Reply:
Updated!